R/metadata.R

```diff
@@ -454,7 +454,18 @@ getInfo <- function(metadata, field=c("station", "custom")) {
   slotname <- field
   
   # Return the metadata stored in that slot
-  slot(metadata, slotname)
+  retVal <- slot(metadata, slotname)
+  #rloadest doesn't accept the default output format
+  #could break this out to a new function
+  if(!is.null(unit.format) && unit.format == "rloadest") {
```

Comment:
i like this for now. someday might be able to move to a more generic option that probably involves adding to the `loadflex:::unit.conversions` table,, but this is completely adequate for the near future